osquery/tables/system/darwin/time_machine_destinations.cpp

```diff
@@ -0,0 +1,68 @@
+// Copyright 2004-present Facebook. All Rights Reserved.
+
+#include <string>
+
+#include <boost/property_tree/ptree.hpp>
+
+#include <osquery/filesystem.h>
+#include <osquery/tables.h>
+
+#include "osquery/core/conversions.h"
+
+namespace pt = boost::property_tree;
+
+namespace osquery {
+namespace tables {
+
+const std::string kTimeMachinePrefs =
+    "/Library/Preferences/com.apple.TimeMachine.plist";
+const std::string kDestinationKey = "Destinations";
```

Comment:
Why so many global const strings, can you just write them inline? Typically we only place configurables outside of methods/classes.